packages/headless/src/controllers/generated-answer/headless-generated-answer.ts

```diff
@@ -48,11 +45,11 @@ export interface GeneratedAnswer extends Controller {
    */
   retry(): void;
   /**
-   * Determines if the generated answer was liked, or upvoted by the end user.
+   * Marks the generated answer as relevant to the end user.
    */
   like(): void;
   /**
-   * Determines if the generated answer was disliked, or downvoted by the end user.
+   * Marks the generated answer as not relevant to the end user.
```

Comment:
```suggestion
   * Marks the generated answer as irrelevant to the end user.
```
I'm not sure about this modification given the dislike action will open the modal and the user can specify what they dislike. Irrelevant is one of the choices but there are others.

Reply:
I tried rephrasing the description to better convey the idea that invoking `like()` does modify the generated answer state to mark it as relevant. The previous description was more like a state retrieval function.

At first, I was using `helpful/not helpful` but went with `relevant/irrelevant` because it was already being used in other descriptions. The idea was to remain consistent.

Reply:
I love @srosenberg-coveo suggestion to use "the generated answer does not meet the user expectations" to convey the idea of an irrelevant answer. It also solves the conflict between the broad meaning of "irrelevant" versus the specific option in the list.

What would you think of something like this?

```typescript
/**
 * Indicate that the generated answer meets the user expectations.
 */
like(): void;
/**
 * Indicate that the generated answer does not meet the user expectations.
 */
dislike(): void;
```

Reply:
love it